src/common/components/header/header.tsx

```diff
@@ -1,7 +1,55 @@
 import React from "react";
+import { Link } from "react-router-dom";
 
+import "./header.scss";
+import logo from "../../../common/assets/Logo-red.png";
+
+const navItems = [
+  {
+    id: 1,
+    title: "Home",
+    path: "/",
+    nName: "navbar--item",
+    sName: "sidebar--item",
```

Comment:
It seems that the sName key is not used, so it could be removed.
`nName` does not provide much information about what the value is. I see that it is used for the className, so a better naming could be something like `className`.
But even so, all the items have the same class, this property could be removed as well and passed its value directly to className.

Reply:
Resolved it. I shall use 1 line of code instead of 6 where I can, from now on. :D